### PR TITLE
vpn gateway vpc ref added

### PR DESCRIPTION
**Pull request by BShrivastav**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000
https://jiracloud.swg.usma.ibm.com:8443/browse/UI-22652
Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/is_vpn_gateway.html.markdown [hunk 1]
```diff
@@ -89,4 +89,14 @@ In addition to all argument references listed, you can access the following attr
...
+- `vpc` - (String) 	The VPC this VPN server resides in.
+  Nested scheme for `vpc`:
+  - `crn` - (String) The CRN for this VPC.
+  - `deleted` - (List) 	If present, this property indicates the referenced resource has been deleted and provides some supplementary information.
+	  Nested scheme for **deleted**:
+		- `more_info` - (String) Link to documentation about deleted resources.
+  - `href` - (String) - The URL for this VPC
+  - `id` - (String) - The unique identifier for this VPC.
+  - `name` - (String) - The unique user-defined name for this VPC.
+  - `resource_type` - (String) - The resource type.
```
Comment (deepaksibm) [inline website/docs/d/is_vpn_gateway.html.markdown:101]: resource_type is not within vpc

File: website/docs/d/is_vpn_gateways.html.markdown [hunk 2]
```diff
@@ -56,6 +56,16 @@ In addition to all argument reference list, you can access the following attribu
...
+  - `vpc` - (String) 	The VPC this VPN server resides in.
+  `Nested scheme for `vpc`:
+  - `crn` - (String) The CRN for this VPC.
+  - `deleted` - (List) 	If present, this property indicates the referenced resource has been deleted and provides some supplementary information.
+	  Nested scheme for **deleted**:
+		- `more_info` - (String) Link to documentation about deleted resources.
+  - `href` - (String) - The URL for this VPC
+  - `id` - (String) - The unique identifier for this VPC.
+  - `name` - (String) - The unique user-defined name for this VPC.
+  - `re`source_type` - (String) - The resource type.
```
Comment (deepaksibm) [inline website/docs/d/is_vpn_gateways.html.markdown:68]: resource_type is not within vpc. should be removed

File: website/docs/r/is_vpn_gateway.html.markdown [hunk 3]
```diff
@@ -80,6 +80,18 @@ In addition to all argument reference list, you can access the following attribu
...
+- `vpc` - (String) 	The VPC this VPN server resides in.
+  Nested scheme for `vpc`:
+  - `crn` - (String) The CRN for this VPC.
+  - `deleted` - (List) 	If present, this property indicates the referenced resource has been deleted and provides some supplementary information.
+	  Nested scheme for **deleted**:
+		- `more_info` - (String) Link to documentation about deleted resources.
+  - `href` - (String) - The URL for this VPC
+  - `id` - (String) - The unique identifier for this VPC.
+  - `name` - (String) - The unique user-defined name for this VPC.
+  - `resource_type` - (String) - The resource type.
```
Comment (deepaksibm) [inline website/docs/r/is_vpn_gateway.html.markdown:92]: resource_type not within vpc

**Comment by BShrivastav** [comment]
vpn-gateway-1
vpn-gateway-2
vpn-gateway-3
vpn-gateway-4

**Pull request closed without merging**